docs/choosing_clusters.md

```diff
@@ -121,23 +121,30 @@ kubectl config use-context microk8s
 [Minikube](https://github.com/kubernetes/minikube) is what we recommend when
 you're willing to pay some overhead for a more high-fidelity cluster.
 
-Minikube creates a Kubernetes cluster in a VM, and has tons of options for
-customizing the cluster.
+Minikube has tons of options for customizing the cluster. You can choose between
+a VM and a Docker container for running a machine, choose from different
+container runtimes, and more.
+
+Follow these instructions to set up Minikube for use with Tilt:
+
+[**Minikube Setup Instructions**](https://github.com/tilt-dev/minikube-local)
```

Comment:
consider the `.attached-above` class on this link for emphasis + spacing (e.g. https://github.com/tilt-dev/tilt.build/blob/e69f39aaa38af81ddcf9ae6b4030d2e85296715e/docs/example_python.md#L30 )

Reply:
done